### PR TITLE
Add support for PHP 8, Symfony 5 and collision 4/5

### PR DESCRIPTION
**Pull request by ostrolucky**

### CODE REVIEW

**Comment by nunomaduro** [comment]
Can you add GitHub actions, just we have in the main Collision repository? So we can deploy this tests being run on CI? Thank you for helping.

**Comment by ostrolucky** [comment]
done

File: .github/tests.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+on: ['push', 'pull_request']
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: true
+            matrix:
+                php: [7.3 7.4, 8.0]
```
Comment (nunomaduro) [inline .github/tests.yml]: This is not correct.

Comment (ostrolucky) [reply .github/tests.yml]: can you elaborate? which versions do you want there?

Comment (nunomaduro) [reply .github/tests.yml]: Actually, can I transfer this repository to you? I don't plan to maintain this anymore.

Comment (ostrolucky) [reply .github/tests.yml]: Me neither, I'm just random contributor here, I don't even use this package :( I wanted to test it out and saw it's not installable so I just fixed it.

**Pull request merged**